### ISSUE TITLE
The iPhone does not recognising which application to share the file that was send with opendrop cli

### ISSUE THREAD

**Issue by boaz-warshawsky** [seemoo-lab/opendrop#56]
I am having a problem that does not happen if I sending a file "normally" via the mac AirDrop in Finder. 

When I am sending my file via the finder AirDrop, when the file receives on the iPhone, the iPhone offers me to open the exact application to which the file is targeted to.
When I am sending the file with `opendrop` cli the iPhone telling me that the shared file requires another app from app store:
![IMG_0004](https://user-images.githubusercontent.com/71062516/97612852-fbc5f280-1a28-11eb-9d05-931c0eb542ff.PNG)

My cli:
`opendrop send -r 0 -f /Users/boazwarshawsky/Downloads/m200_bundle_1.0.6.1.app.only.fw`

**Comment by boaz-warshawsky** [comment]
The expected message I should get, and that I am getting when sending it via the finder is that:
![IMG_0005](https://user-images.githubusercontent.com/71062516/97613295-8f97be80-1a29-11eb-9f12-1496bc6c4397.PNG)

**Comment by schmittner** [comment]
You are correct. OpenDrop does not support all file types properly. This is due to Apple's proprietary [UTI](https://developer.apple.com/library/archive/documentation/Miscellaneous/Reference/UTIRef/Articles/System-DeclaredUniformTypeIdentifiers.html) system which we currently only support in a very limited fashion here:
https://github.com/seemoo-lab/opendrop/blob/a309c1946e30c572cd97c6324858c5f5b07bc70d/opendrop/util.py#L52
Unfortunately, we do not have the capacity to extend this list and make it work with all content types.

**Comment by schmittner** [comment]
One could probably generate a valid MIME–UTI mapping by using Apple's [`UTType`](https://developer.apple.com/documentation/uniformtypeidentifiers/uttype) class.
